<ui version="4.0" >
 <class>qslConf</class>
 <widget class="QDialog" name="qslConf" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QSL Config</string>
  </property>
  <widget class="QPushButton" name="ButtonExit" >
   <property name="geometry" >
    <rect>
     <x>64</x>
     <y>296</y>
     <width>461</width>
     <height>34</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>214</red>
         <green>229</green>
         <blue>232</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>214</red>
         <green>229</green>
         <blue>232</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>214</red>
         <green>229</green>
         <blue>232</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string>Save / Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>164</x>
     <y>84</y>
     <width>173</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Upload Dataset </string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLotwShort" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>128</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string>Default</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLotwLong" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>204</y>
     <width>73</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string>Long</string>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>164</x>
     <y>132</y>
     <width>341</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>CALL, BAND, MODE, QSO_DATE, TIME_ON</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>164</x>
     <y>152</y>
     <width>391</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>QSL_RCVD, QSL_RDATE, QSL_SENT, QSLSDATE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>164</x>
     <y>204</y>
     <width>336</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>default + CQZ, ITUZ, DXCC, GRIDSQUARE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>164</x>
     <y>224</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>MY_NAME, MY_QTH</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16" >
   <property name="geometry" >
    <rect>
     <x>68</x>
     <y>84</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>30</red>
         <green>62</green>
         <blue>161</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>30</red>
         <green>62</green>
         <blue>161</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>146</red>
         <green>145</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Lotw</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>checkBoxLotwLong</tabstop>
  <tabstop>checkBoxLotwShort</tabstop>
  <tabstop>ButtonExit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
